### ISSUE TITLE
[CVE Report]: idealista/prom2teams:3.2.2

### ISSUE THREAD

**Issue by jiaqiluo** [idealista/prom2teams#269]
Hi, could you update the image to address the CVE found by `trivy`, especially the `CVE-2021-36222` which has a high severity?

```
> trivy image idealista/prom2teams:3.2.2
2021-08-02T11:01:40.886-0700	INFO	Detected OS: alpine
2021-08-02T11:01:40.886-0700	INFO	Detecting Alpine vulnerabilities...
2021-08-02T11:01:40.891-0700	INFO	Number of language-specific files: 0

idealista/prom2teams:3.2.2 (alpine 3.14.0)
==========================================
Total: 2 (UNKNOWN: 1, LOW: 0, MEDIUM: 0, HIGH: 1, CRITICAL: 0)

+-----------+------------------+----------+-------------------+---------------+---------------------------------------+
|  LIBRARY  | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |                 TITLE                 |
+-----------+------------------+----------+-------------------+---------------+---------------------------------------+
| apk-tools | CVE-2021-36159   | UNKNOWN  | 2.12.5-r1         | 2.12.6-r0     | -->avd.aquasec.com/nvd/cve-2021-36159 |
+-----------+------------------+----------+-------------------+---------------+---------------------------------------+
| krb5-libs | CVE-2021-36222   | HIGH     | 1.18.3-r1         | 1.18.4-r0     | krb5: sending a request containing    |
|           |                  |          |                   |               | a PA-ENCRYPTED-CHALLENGE padata       |
|           |                  |          |                   |               | element without using FAST...         |
|           |                  |          |                   |               | -->avd.aquasec.com/nvd/cve-2021-36222 |
+-----------+------------------+----------+-------------------+---------------+---------------------------------------+
```

**Comment by jiaqiluo** [comment]
bump

**Comment by vicsufer** [comment]
Hi @jiaqiluo new images were generated along with release 3.2.3:
- [latest](https://hub.docker.com/layers/idealista/prom2teams/latest/images/sha256-9e9784eb2177d35688d194c7faf7bf9695012ef99c9ca30cf141489bfd8b39b1?context=explore&tab=layers)
- [3.2.3](https://hub.docker.com/layers/idealista/prom2teams/3.2.3/images/sha256-9e9784eb2177d35688d194c7faf7bf9695012ef99c9ca30cf141489bfd8b39b1?context=explore)

Both `apk-tools` and `krb5-libs` have been updated.

**Comment by jiaqiluo** [comment]
@vicsufer  Thank you for providing the fix!  

I can confirm that the above-mentioned CVEs are fixed in `idealista/prom2teams:3.2.2`

```
> trivy image idealista/prom2teams:3.2.3

2021-09-06T20:28:46.643-0700	INFO	Need to update DB
2021-09-06T20:28:46.643-0700	INFO	Downloading DB...
23.46 MiB / 23.46 MiB [--------------------------------------------------------------------------------------------------------------------------------------] 100.00% 48.72 MiB p/s 0s
2021-09-06T20:28:51.120-0700	INFO	Detected OS: alpine
2021-09-06T20:28:51.120-0700	INFO	Detecting Alpine vulnerabilities...
2021-09-06T20:28:51.121-0700	INFO	Number of language-specific files: 0

idealista/prom2teams:3.2.3 (alpine 3.14.2)
==========================================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```
